nrpe_exporter.go

```diff
@@ -112,20 +132,22 @@ func handler(w http.ResponseWriter, r *http.Request, logger log.Logger) {
 		http.Error(w, "Command parameter is missing", 400)
 		return
 	}
+	sslParam := params.Get("ssl")
+	ssl := (sslParam == "1" || sslParam == "true" || sslParam == "yes")
```

Comment:
Only accept "true", that's consistent with the blackbox exporter's debug option.